tests/test_servicex_dataset.py

```diff
@@ -527,3 +526,82 @@ def test_transform_request():
                               "(lambda (list e) (dict (list 'lep_pt') " \
                               "(list (subscript e 'lep_pt')))))"
         cache.close()
+
+
+@pytest.mark.asyncio
+async def test_use_of_cache_ignore_cache(mocker):
+    """ Do we pick up the cache on the second request for the same transform? """
+    servicex = AsyncMock()
```

Comment:
Could you pull these mocks out into a [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html)? It will make this test cleaner and easier to read... also we will hopefully need these for other tests